common/script/fns/cron.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(user, options) {
   }
   user.auth.timestamps.loggedin = new Date();
   user.lastCron = now;
+  if (_.isFinite(+user.preferences.timezoneOffset)) {
```

Comment:
But sorry, I should have said, I'll change it to `if (timezoneOffset in user.preferences)`. I don't feel strongly about my current method and I'm happy to use yours.